app/src/main/java/de/tum/in/tumcampusapp/adapters/LectureListSelectionAdapter.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.tumcampusapp.adapters;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.CursorAdapter;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.managers.CalendarManager;
+
+public class LectureListSelectionAdapter extends CursorAdapter implements CompoundButton.OnCheckedChangeListener {
+    private int appWidgetId;
+    private CalendarManager calendarManager;
+
+    public LectureListSelectionAdapter(Context context, Cursor c, boolean autoRequery, int appWidgetId) {
+        super(context, c, autoRequery);
+        this.appWidgetId = appWidgetId;
+        this.calendarManager = new CalendarManager(context);
+    }
+
+    @Override
+    public View newView(Context context, Cursor cursor, ViewGroup parent) {
+        return LayoutInflater.from(context).inflate(R.layout.list_timetable_configure_item, parent, false);
+    }
+
+    @Override
+    public void bindView(View view, Context context, Cursor cursor) {
+        CheckBox checkBox = (CheckBox) view.findViewById(R.id.timetable_configure_item);
+        checkBox.setChecked(cursor.getInt(cursor.getColumnIndex("is_on_blacklist")) == 0);
+        checkBox.setText(cursor.getString(cursor.getColumnIndex("title")));
+        checkBox.setOnCheckedChangeListener(this);
```

Comment:
maybe use setTag to save the lecture "id" and then simplify the lecure title for the text view